sass/includes/_featured-records.scss

```diff
@@ -1,4 +1,6 @@
 .featured-records {
+  margin-bottom: 4rem;
```

Comment:
Hey @jamesbiggs,

You're right, it was my mistake, this template is not used where I thought it was. The templates/ styling class names are too similar. I think 4rem here is little too large, could we use 2rem? That's more inkeeping with the rest of the spacing used within the streamfield elements.